src/legacy/core_plugins/kibana/common/field_formats/types/__tests__/url.js

```diff
@@ -189,5 +189,29 @@ describe('UrlFormat', function () {
       expect(converter('../foo/bar', null, null, parsedUrl))
         .to.be('<span ng-non-bindable><a href="http://kibana.host.com/nbc/app/../foo/bar" target="_blank" rel="noopener noreferrer">../foo/bar</a></span>');
     });
+
+    it('should support multiple types of urls w/o basePath', function () {
+      const url = new UrlFormat();
+      const parsedUrl = {
+        origin: 'http://kibana.host.com',
+        pathname: '/app/kibana',
+      };
+      const converter = url.getConverterFor('html');
+
+      expect(converter('10.22.55.66', null, null, parsedUrl))
```

Comment:
@ppisljar  It's not related to this issue but not sure that the following URL is valid: `http://kibana.host.com/app/10.22.55.66` Do you know how we should process IP addresses?

Reply:
it should work the way it did on 6.x, unless that behavior is obviously wrong.

Reply:
It works such way now